libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -234,11 +233,21 @@ AP_GPS_UBLOX::_verify_rate(uint8_t msg_class, uint8_t msg_id, uint8_t rate) {
                 _cfg_needs_save = true;
             }
             break;
+        case MSG_PVT:
+            if(rate == RATE_PVT) {
+                _unconfigured_messages &= ~CONFIG_RATE_PVT;
+            } else {
+                _configure_message_rate(msg_class, msg_id, RATE_PVT);
+                _unconfigured_messages |= CONFIG_RATE_PVT;
+                _cfg_needs_save = true;
+            }
```

Comment:
Ummmm, valid point, that is a possibility unfortunately. I was attempting to avoid the situation where we request a config then immediately disable it, and then because we changed a setting on the GPS unit we immediately decide to save the configuration.

Easiest fix is that I could restore a series of rapid requests until we've requested PVT/POSLLH/VELNED the first time. I never really liked having those though, and was excited to delete them...